### ISSUE TITLE
Native Notebooks - Outline contents generated out of order

### ISSUE THREAD

**Issue by claudiaregio** [microsoft/vscode#114220]
![image](https://user-images.githubusercontent.com/19276747/104356095-64692b00-54c0-11eb-8b5a-2aa94b0ea419.png)

**Comment by jrieken** [comment]
@claudiaregio what order is set? please check the overflow menu for that

<img width="508" alt="Screenshot 2021-01-13 at 08 52 11" src="https://user-images.githubusercontent.com/1794099/104422144-a2c42000-557c-11eb-8d1a-7791cf38efbd.png">

**Comment by claudiaregio** [comment]
Thanks @jrieken, it was sorted by category. Shouldn't the default for notebooks be position though?

**Comment by jrieken** [comment]
Indeed. The default is "category" which doesn't make much sense, let me fix this
